test/compilable/cpp.d

```diff
@@ -0,0 +1,19 @@
+
+// Test C++ features.
+
+version(Windows)
+{
+    version(D_LP64)
```

Comment:
You can just use CRuntime_Microsoft instead.

Reply:
Oh yeah... we have those now huh ;)